plugin/adsk/scripts/AETemplateHelpers.py

```diff
@@ -207,5 +229,4 @@ def ProxyShapeFilePathRefresh(filePathAttr):
                     debugMessage('  User confirmed reload action, calling UsdStage.Reload()')
                     proxyStage.Reload()
     except Exception as e:
-        debugMessage('ProxyShapeFilePathRefresh() - Error: %s' % str(e))
-        pass
+        print('ProxyShapeFilePathRefresh() - Error: %s' % str(e))
```

Comment:
Why is this a plain print now? The user cannot do anything about the error.

Reply:
The alternative is that if the button fails for any reason, it does so silently. I can remove the print, but I treat differently random errors printed without user action vs error triggered by user action. This error is rare enough that I don't really mind either way, so I'll just revert it.